lib/election.js

```diff
@@ -417,32 +408,48 @@ function _createEventManifest(ledgerNode, blockHeight, callback) {
  *          completes.
  */
 function _recommendElectors(ledgerNode, voter, electors, manifest, callback) {
-  // TODO: use given parameters and blockchain to determine a set of
-  // electors to recommend for the next block
-
-  // FIXME: include all current electors (should filter this by those
-  //   that voted in the last election so we don't recommend those that
-  //   aren't participating)
-  let recommendedElectors = electors.map(elector => ({id: elector.id}));
+  let recommendedElectors = [];
 
   // TODO: implement option to add all nodes that sent in events to the
   // previous block to the elector pool that can be contacted
 
   async.auto({
+    config: callback => _getLatestConfig(ledgerNode, callback),
     latestBlock: callback => ledgerNode.storage.blocks.getLatest(callback),
-    getEventPeers: ['latestBlock', (results, callback) => {
+    latestVoters: ['latestBlock', (results, callback) => {
+      // there are no election results in the genesis block
+      const voters =
+        _.get(results, 'latestBlock.eventBlock.block.electionResults', [])
+          .map(v => v.voter);
+      recommendedElectors.push(...voters);
+      callback();
+    }],
+    eventPeers: ['latestVoters', (results, callback) => {
       if(!results.latestBlock.eventBlock.block) {
         return callback(null, []);
       }
       const events = results.latestBlock.eventBlock.block.event || [];
       // TODO: _getEventPeers returns an array...
```

Comment:
What's TODO here? This is just a statement of truth.

Reply:
I was wondering the same thing, I'll remove it.